### PR TITLE
CMSIS-NN: Fix bug in max pool for MVEI

### PR DESCRIPTION
**Pull request by mansnils**
This is a fix for: https://github.com/ARM-software/CMSIS_5/issues/1131

### CODE REVIEW

**Comment by mansnils** [comment]
@felix-johnny Can you review this?

**Comment by felix-johnny** [comment]
@mansnils could you add a TC as well for this?

**Comment by mansnils** [comment]
@felix-johnny Ok will do

File: CMSIS/NN/Tests/UnitTest/TestCases/TestData/maxpooling_7/config_data.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#pragma once
+#define MAXPOOLING_7_OUT_CH 1
+#define MAXPOOLING_7_IN_CH 1
+#define MAXPOOLING_7_INPUT_W 4
+#define MAXPOOLING_7_INPUT_H 2
+#define MAXPOOLING_7_DST_SIZE 2
+#define MAXPOOLING_7_INPUT_SIZE 8
+#define MAXPOOLING_7_INPUT_OFFSET -128
+#define MAXPOOLING_7_OUTPUT_OFFSET 0
+#define MAXPOOLING_7_OUT_ACTIVATION_MIN 0
```
Comment (felix-johnny) [inline CMSIS/NN/Tests/UnitTest/TestCases/TestData/maxpooling_7/config_data.h:29]: Looking at the activation min/max all other test cases have the int8 min/max values.. And its good that this one has 0 as min.. but, would be good to have non-127 as max as well..could you update that either in this TC or in another one..?
 Also, could you make the output offset non-zero? all have zero right now.

Comment (mansnils) [reply CMSIS/NN/Tests/UnitTest/TestCases/TestData/maxpooling_7/config_data.h:29]: Ok, removing offsets for the pooling tests as agreed.

**Pull request merged**